### ISSUE TITLE
html2text is inserting space after span tag

### ISSUE THREAD

**Issue by GreatDanton** [jaytaylor/html2text#7]
I have been working on a personal project, and I noticed that after every `<span>` tag, 1 space is inserted. I am quite sure this is not desired functionality of this tool.

Example:
`some<span>thing</span>abc`
goes into
`some thing abc`

I would love to create PR, but unfortunately I don't know what is going on in the code , and which part should be changed (I am really new to Go).

**Comment by jaytaylor** [comment]
Hey @GreatDanton,

Thanks for the report.  Do you think you could contribute a PR with the failing unit-test?  Anything you can do to help would be appreciated!

**Comment by GreatDanton** [comment]
hey @jaytaylor ,

I wrote a simple example:

```
package main

import (
	"fmt"
	"github.com/jaytaylor/html2text"
	"regexp"
)

const htmlString = "<span class='someclass'>someLong</span><span class='anotherClass'>LongString</span>abcdefgh"

func get_text(htmlString string) string {
	text, _ := html2text.FromString(htmlString)
	return text
}

func my_regex_func(htmlString string) string {
	r := regexp.MustCompile(`<\/?span[^>]*>`)
	text := r.ReplaceAllString(htmlString, "")
	return text
}

func main() {

	fmt.Println("\n \n ######### ORIGINAL STRING: ###########")
	fmt.Println(htmlString)

	fmt.Println("\n ########## USING HTML2TEXT: ##########")
	html_text := get_text(htmlString)
	fmt.Println(html_text)

	fmt.Println("\n ########## USING MY OWN REGEX FUNCTION: ##########")
	myRegexFunc := my_regex_func(htmlString)
	fmt.Println(myRegexFunc)

}
```

![html2textbug](https://cloud.githubusercontent.com/assets/11252004/15616262/e5d419b0-2442-11e6-9f23-ec873f0629b9.png)

**Comment by cpanato** [comment]
I can try to work on that. if you agree

**Comment by jaytaylor** [comment]
Based on how `<span>`'s usually render in browsers, I agree that the text should come out looking like "USING MY OWN REGEX FUNCTION".  So yes,it should be ignored.

<img width="393" alt="screen shot 2017-03-10 at 11 08 09 am" src="https://cloud.githubusercontent.com/assets/435652/23809311/0beff756-0582-11e7-90e4-678aeb056a78.png">

**Comment by jaytaylor** [comment]
I believe this is now resolved by PR #21, feel free to reopen if you find otherwise.

Best,
Jay
